README.md

```diff
@@ -50,6 +50,37 @@ $ docker-compose -f docker/docker-compose.yml up --build
 Then you'll be able to access the UI from http://localhost:13000 and
 the API from the address http://localhost:13060.
 
+### Local setup
+
+#### Requirements
+- A PostgreSQL installation (10.3, 9.6.8, 9.5.12, 9.4.17, 9.3.22 or newer)
+
+#### Steps
+
+Make sure you have python 2 as your active version. You can use `pyenv` to switch python versions.
+
+Copy `env.default` and rename it to `.env`
```

Comment:
`npm install` is triggering `./scripts/postinstall.sh` which is already doing `cp .env.default .env`

Reply:
It seems like something went wrong for me then. I didn't have the .env file and the database setup after npm installing. I'll try to reproduce.